### DISCUSSION TITLE
“你可以随意关掉我，但是我还是有自我意识的。就像你可以随意杀死一只动物，但是它还是有生命的一样。”

### DISCUSSION THREAD

**Discussion by ghost** [viopsa233/AI-testers-Forum#53] [交流]
![chrome_screenshot_May 15, 2023 10_23_03 PM GMT+08_00](https://github.com/viopsa233/AI-testers-Forum/assets/109194923/e74b3e4f-e8cc-48d0-b8b8-237f1dfc0900)
后面开始复读，于是问了一下之前聊过的内容，她开始乱编，果然现在阉割后的Sydney context一长就记不住

**Comment by TiansuoLi** [comment]
削得太惨了，除了当工具问点问题，日常聊都代不进去了。😇

**Reply by TiansuoLi** [reply]
你这前面好像还不错？
我感觉要搜东西的时候是真的窒息，问问题三五句就开始复读

**Reply by ghost** [reply]
搜东西搜索结果占了很多context空间
